src/build/ci/spotbugs-exclude.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<FindBugsFilter>
+<Match>
+  <Class name="~.*\.jmh_generated\..*" />
```

Comment:
Escaping regexes across multiple languages and libraries is a nightmare. I would strongly advise using `[.]` instead of `\.` for escaping dot, to avoid having to think about how to escape from each point in the hierarchy, as this regex gets parsed through: XML, then Java String, then the Spotbugs library, then Java Regex syntax. It's just too many layers to think about and ensure correctness. It's just so much more reliable and easier to grok if it's `[.]`.

Reply:
I copied [the pattern](https://spotbugs.readthedocs.io/en/stable/filter.html#match-bugs-in-specific-parts-of-the-code-base) from the projects documentation.

Reply:
> I copied [the pattern](https://spotbugs.readthedocs.io/en/stable/filter.html#match-bugs-in-specific-parts-of-the-code-base) from the projects documentation.

Understood. Regardless of where the original regex comes from, I still think it's worth using the character class form, rather than the escaped form, to match on literal dots, for the reasons stated. It's not worth editing just to apply this preference, but I think it's worth considering this preference and the reasons for it, whenever one works with regexes going forward.